en/docs/guides/login/session-persistence.md

```diff
@@ -0,0 +1,131 @@
+# Authentication Session Persistence
+
+This guide walks you through the process of enabling session persistence for sessions. This is useful when the **Remember me** option is selected while logging into either the service provider or the WSO2 Identity Server.
```

Comment:
`while logging into either the service provider or the WSO2 Identity Server.` ---> what does this mean?

Reply:
service provider -> Example: Pickup dispatch application
WSO2 Identity Server -> Management console / MyAccount